ckanext/unaids/blueprints/unaids_dataset_transfer.py

```diff
@@ -31,9 +36,9 @@ def process_dataset_transfer(dataset_id):
         'org_to_allow_transfer_to': None
     })
     toolkit.get_action('package_update')({
+        'user': _admin_user()['name'],
```

Comment:
I have described the reason why it's needed here: https://github.com/ckan/ckan/issues/6779
An easier workaround to this bug is to pass empty `'user'` value like:
```
toolkit.get_action('package_update')({
        'user': ''
        'model': model,
        'session': model.Session,
      'ignore_auth': True
```